index.html

```diff
@@ -220,6 +219,75 @@ <h3>Skills</h3>
 			</div>
 		</section>
 
-		<script src="main.js"></script>
-	</body>
-</html>
+        <section id="contact">
+			<div class="pri-text">
+				<p class="contact-para">
+					I'm always interested in hearing about new projects, so if you'd like
+					to chat please get in touch.
+				</p>
+			</div>
+			<form action="https://formspree.io/f/xpznwejk" method="post" class="form">
+				<fieldset>
+					<label class="fullN"
+						><input
+							type="text"
+							name="fullname"
+							id="fullname"
+							maxlength="30"
+							placeholder="Full name"
+							required
+						/>
+					</label>
+
+					<label class="email"
+						><input
+							type="email"
+							name="email"
+							id="email"
+							placeholder="Email address"
+							required
+						/>
+					</label>
+					<label class="msg">
+						<textarea
+							name="msg"
+							id="msg"
+							cols="30"
+							rows="10"
+							maxlength="500"
+							minlength="30"
+							placeholder="Enter text here"
+							required
+						></textarea>
+					</label>
+					<button class="btn" type="submit">Get in Touch</button>
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly follow the template when it comes to button interaction styles 👇 

## Your design (on hover)
![image](https://user-images.githubusercontent.com/50721479/213875127-7379b04c-accb-499d-b8a2-3fd91ce2f2a2.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/213875136-3c3ddb36-d6be-4dc6-abb9-c5beed0fc9a6.png)